### ISSUE TITLE
Installing the extension

### ISSUE THREAD

**Issue by tonyfast** [jupyter/telemetry#22]
I was trying to install the labextension locally and it seems like the `package.json` is out of date or incomplete.  Maybe the javascript part of the extension can be tested on circle, too.

**Comment by Zsailer** [comment]
Yeah, the JS side of this package hasn't been developed (yet). Right now, this is just a Python library. It's coming soon though! 😃

**Comment by tonyfast** [comment]
Cool.  What is the minimal thing I need to do start logging events?

**Comment by Zsailer** [comment]
Here are instructions to getting telemetry working with jupyter/notebook. 

1. Clone and install this repo.
2. Fetch [this branch](https://github.com/yuvipanda/notebook/tree/feat/telemetry) of notebook.
3. Add this configuration to your `jupyter_notebook_config.py` (or however you store/load your config). 
    ```python
     import logging

     # List where you want your events to go.
     eventlog_filepath = 'events.log'
     c.EventLog.handlers = [
         logging.FileHandler(eventlog_filepath)
     ]

     # List which events get logged.
     c.EventLog.allowed_schemas = [
        'eventlogging.jupyter.org/notebook/contentsmanager-actions'
     ]
    ```
4. Start jupyter notebook and navigate directories/open+close ntoebooks. 

The current implementation logs events from the contents service (we still need to develope schemas for other services).

I'll work on the README today to give examples on how to integrate this library with other tools.
